tests/apm/commandFailed-001.phpt

```diff
@@ -0,0 +1,55 @@
+--TEST--
+APM: commandFailed callback
+--FILE--
+<?php
+include dirname(__FILE__) . '/../utils.inc';
+
+$m = new MongoDB\Driver\Manager("mongodb://localhost:27017");
+cleanup( $m );
+
+class MySubscriber implements MongoDB\Driver\Monitoring\CommandSubscriber
+{
+	public function commandStarted( \MongoDB\Driver\Monitoring\CommandStartedEvent $event )
+	{
+		echo "started: ", $event->getCommandName(), "\n";
+	}
+
+	public function commandSucceeded( \MongoDB\Driver\Monitoring\CommandSucceededEvent $event )
+	{
+	}
+
+	public function commandFailed( \MongoDB\Driver\Monitoring\CommandFailedEvent $event )
+	{
+		echo "failed: ", $event->getCommandName(), "\n";
+		echo "- getError() returns an object: ", is_object( $event->getError() ) ? 'yes' : 'no', "\n";
+		echo "- getError() returns an MongoDB\Driver\Exception\Exception object: ", $event->getError() instanceof MongoDB\Driver\Exception\Exception ? 'yes' : 'no', "\n";
```

Comment:
Is that preceding "getError() returns an object" test because `instanceof` would yield a fatal error if it's lval was not an object?